project-jee8-webapp/src/main/java/fr/pantheonsorbonne/ufr27/miage/jms/InfoCentrePublisher.java

```diff
@@ -66,11 +66,12 @@ public void close() throws IOException {
 	public String prepareBulletinToPublish(List<Train> listTrains) {
 		StringBuilder message = new StringBuilder();
 		message.append("------------------------INFO TRAINS---------------------\n");
+		// this should be converted in DTO and sent as XML or Json through JMS
```

Comment:
sending text message makes the filtering of message complicated at the other end. Please use a proper serialization instead